### PR TITLE
Input Authenticator Page

### PR DESCRIPTION
**Pull request by amir-budaychiev**
Related with issue #152

### CODE REVIEW

File: app/views/security/security_authentication.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div id="security-auth">
```
Comment (mod) [inline app/views/security/security_authentication.html.erb]: rename the page otp.html.erb

File: app/views/security/security_authentication.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <div class="row">
+    <div class="col-12">
+      <div class="top-section">
+        <p>
+          Security authentication
```
Comment (mod) [inline app/views/security/security_authentication.html.erb]: 2Factor Challenge

File: app/views/security/security_authentication.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        </p>
+      </div>
+    </div>
+    <div class="col-6">
+      <div class="left-section">
+        <label>
+          Authenticator Code
```
Comment (mod) [inline app/views/security/security_authentication.html.erb]: Security code

File: config/routes.rb [hunk 4]
```diff
@@ -9,7 +9,8 @@
   root to: 'index#index', as: :index
 
   post  'phones/verification', to: 'phones#verify'
-  get   'security',     to: 'security#enable'
+  get   'security',                 to: 'security#enable'
+  get   'security_authentication',  to: 'security#security_authentication'
```
Comment (mod) [inline config/routes.rb]: We can make the route security#otp for now
but this will certainly move in the login process

**Comment by amir-budaychiev** [comment]
@mod Done

**Pull request closed without merging**